test/test_tree_1d_euler.jl

```diff
@@ -91,35 +91,40 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_1
   @trixi_testset "elixir_euler_sedov_blast_wave.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_sedov_blast_wave.jl"),
       l2   = [1.250005061244617, 0.06878411345533507, 0.9264328311018613],
-      linf = [2.9766770877037168, 0.16838100902295852, 2.6655773445485798])
+      linf = [2.9766770877037168, 0.16838100902295852, 2.6655773445485798],
+      coverage_override = (maxiters=6,))
   end
 
   @trixi_testset "elixir_euler_sedov_blast_wave_pure_fv.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_sedov_blast_wave_pure_fv.jl"),
       l2   = [1.075075094036344, 0.06766902169711514, 0.9221426570128292],
-      linf = [3.3941512671408542, 0.16862631133303882, 2.6572394126490315])
+      linf = [3.3941512671408542, 0.16862631133303882, 2.6572394126490315],
+      coverage_override = (maxiters=10^5, tspan=(0.0, 0.1)))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
Because this is one of the AMR tests that need to run a bit longer to cover all the AMR code. In this case, we don't need a certain amount of time steps but a certain physical time to cover everything.